### PR TITLE
gui-apps/rofi-wayland: new package v1.7.5

### PR DESCRIPTION
**Pull request by rahul-singh-sandhu**
Signed-off-by: Rahul Sandhu <rahul@sandhuservices.dev>

### CODE REVIEW

File: gui-apps/rofi-wayland/manifest.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (thesamesam) [inline gui-apps/rofi-wayland/manifest.xml:1]: Drop newline.

File: gui-apps/rofi-wayland/manifest.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
```
Comment (thesamesam) [inline gui-apps/rofi-wayland/manifest.xml:3]: https

File: gui-apps/rofi-wayland/manifest.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+		<email>rahul@sandhuservices.dev</email>
+		<name>Rahul Singh Sandhu</name>
+	</maintainer>
+  <use>
+    <flag name="drun">Desktop file mode</flag>
+    <flag name="window">Window switcher mode</flag>
+    <flag name="check">Build and run libcheck-based tests</flag>
+    <flag name="wayland">Build with wayland support</flag>
+  </use>
+</pkgmetadata>
```
Comment (thesamesam) [inline gui-apps/rofi-wayland/manifest.xml:15]: for a bonus, add a github remote-id?

File: gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild [hunk 4]
```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
```
Comment (thesamesam) [inline gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild:2]: Blank line here please:
```suggestion
# Distributed under the terms of the GNU General Public License v2

```

File: gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+EAPI=8
```
Comment (thesamesam) [inline gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild:3]: And another:
```suggestion
EAPI=8

```

File: gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+KEYWORDS="~amd64 ~x86"
+IUSE="+drun +windowmode test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	sys-devel/bison
+	sys-devel/flex
+	virtual/pkgconfig
+"
+RDEPEND="
```
Comment (thesamesam) [inline gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild:22]: sort

File: gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+SRC_URI="https://github.com/lbonn/rofi/releases/download/${PV}%2Bwayland1/rofi-${PV}+wayland1.tar.xz"
+S="${WORKDIR}"/rofi-"${PV}"+wayland1
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+drun +windowmode test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
```
Comment (thesamesam) [inline gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild:17]: I prefer to put BDEPEND _after_ RDEPEND + DEPEND.

File: gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+DESCRIPTION="A window switcher, run dialog and dmenu replacement"
+HOMEPAGE="https://github.com/lbonn/rofi"
+SRC_URI="https://github.com/lbonn/rofi/releases/download/${PV}%2Bwayland1/rofi-${PV}+wayland1.tar.xz"
+S="${WORKDIR}"/rofi-"${PV}"+wayland1
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+drun +windowmode test"
```
Comment (thesamesam) [inline gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild:14]: Sort ignoring +

File: gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild [hunk 9]
```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+EAPI=8
+inherit meson xdg
+
+DESCRIPTION="A window switcher, run dialog and dmenu replacement"
+HOMEPAGE="https://github.com/lbonn/rofi"
+SRC_URI="https://github.com/lbonn/rofi/releases/download/${PV}%2Bwayland1/rofi-${PV}+wayland1.tar.xz"
+S="${WORKDIR}"/rofi-"${PV}"+wayland1
```
Comment (thesamesam) [inline gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild:9]: ${PV} will never contain spaces, so:
```suggestion
S="${WORKDIR}"/rofi-${PV}+wayland1
```

**Review by thesamesam — CHANGES_REQUESTED** [review]

File: gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild (hunk 9 above)
Comment (N-R-K) [reply gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild]: or
```sh
S="${WORKDIR}/rofi-${PV}+wayland1"
```
works as well (and IMO is better practice).

**Comment by rahul-singh-sandhu** [comment]
@thesamesam done :)

File: gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild (hunk 9 above)
Comment (Schievel1) [reply gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild]: Why though? I mean what Sam suggested. 

I usually do what NRK says or I put every variable in doubles quotes. Is that wrong?

File: gui-apps/rofi-wayland/manifest.xml [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+	<maintainer type="person">
+		<email>rahul@sandhuservices.dev</email>
+		<name>Rahul Singh Sandhu</name>
+	</maintainer>
+  <use>
+    <flag name="drun">Desktop file mode</flag>
+    <flag name="window">Window switcher mode</flag>
+    <flag name="check">Build and run libcheck-based tests</flag>
+    <flag name="wayland">Build with wayland support</flag>
+  </use>
```
Comment (arthurzam) [inline gui-apps/rofi-wayland/manifest.xml:14]: not the same indent (tabs & spaces)

**Review by arthurzam — CHANGES_REQUESTED** [review]

File: gui-apps/rofi-wayland/manifest.xml (hunk 10 above)
Comment (rahul-singh-sandhu) [reply gui-apps/rofi-wayland/manifest.xml:14]: I'll fix that - thanks

**Comment by rahul-singh-sandhu** [comment]
@thesamesam How is it looking?

**Comment by Schievel1** [comment]
If you sign off all those commits, I will do the rest for you, ok?

**Comment by rahul-singh-sandhu** [comment]
Closing because I have pushed it on the gentoo git.

**Pull request closed without merging**